scripts/cfevents/space-guids

```diff
@@ -0,0 +1,5 @@
+export dev=ea450105-895e-4a33-9541-6a0df35ee626
```

Comment:
oof, and should there be a `dr` here as well?

Reply:
Actually, it would probably be worth using `cf space dev --guid` to gather these guids programatically.

Reply:
file removed.   Single space GUID is now gathered in the script where it is needed.